### PR TITLE
Commenting: Adds fullscreen commenting

### LINKED ISSUE #12918: Enhanced / Full Screen Commenting
At the moment the comment input field is restricted to 5 lines of text. Editing a comment or composing a longer comment can, therefore, be fiddly and generally a bad experience. We want to improve this experience and remove the frustration from composing a longer comment.

**Goals:**

 - Allow a full-screen view for comments
 - Provide a better commenting experience

**Colors**

- Background color -> .basicBackground
- Divider -> .divider
- Text (even title) -> .text
- Placeholder text -> .textPlaceholder
- Expand/Collapse icon -> .listIcon
- Send button active -> .primary or .primaryButtonBackground
- Send button inactive -> .listSmallIcon

### PR DESCRIPTION
**Pull request by emilylaguna**
Depends on https://github.com/wordpress-mobile/WordPress-iOS/pull/13111
Fixes #12918 subtask: Add ability to go fullscreen while commenting

**To test:**

1. Enter any comment flow (reader, notification, comments)
2. Enter some text
3. Tap the full screen toggle button
4. Enter some more text
5. Hit the exit fullscreen button
6. Enter some more text
7. Tap the reply/save button

**Video Demos**

| Device | Version | Link |
|---|---|---|
| iPhone 8 | 11.4 | https://www.dropbox.com/s/eowzknqz3u6vsb1/iPhone%208%20-%2011.4%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPhone 8 | 12.4 | https://www.dropbox.com/s/qcly3q1eer355e5/iPhone%208%20-%2012.4%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPhone 8 | 13.4 | https://www.dropbox.com/s/limjqdt2g9k59d1/iPhone%208%20-%2013.3%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPhone 11 Pro Max | 13.3 | https://www.dropbox.com/s/u897sver579actf/iPhone%2011%20Pro%20Max%20-%2013.3%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPad Air | 11.4 | https://www.dropbox.com/s/qgbpl41s1br2z7d/iPhone%20Air%20-%2011.4%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPad Air | 12.4 | https://www.dropbox.com/s/hh9rn88jb5jgfs5/iPhone%20Air%20-%2012.4%20-%20Fullscreen%20Demo.mov?dl=0 |
| iPad Air | 13.3 | https://www.dropbox.com/s/mwxwpieo9bxhgd7/iPhone%20Air%20-%2013.3%20-%20Fullscreen%20Demo.mov?dl=0 |

**PR submission checklist:**

- [X] I have considered adding unit tests where possible.

- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/System/WordPressAppDelegate.swift [hunk 1]
```diff
@@ -858,9 +858,6 @@ extension WordPressAppDelegate {
     private func customizeAppearanceForTextElements() {
         let maximumPointSize = WPStyleGuide.maxFontSize
 
-        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white,
```
Comment (leandroalonso) [inline WordPress/Classes/System/WordPressAppDelegate.swift]: Hmmm. I'm a bit worried about removing that from the `AppDelegate`. We might end affecting some other `UINavigationBar`. Do you think we can keep that here and customize the comments nav bar?

Comment (emilylaguna) [reply WordPress/Classes/System/WordPressAppDelegate.swift]: Unfortunately I was not able to get that to work on older iOS versions (iOS 12 and below). No matter what I did the title color would always be white, and removing this line is the only thing that worked. 

I went through a number of flows, and didn't see any changes anywhere. 

Now that I think about it I should have made this its own separate PR so we can more easily determine if it's something that we want to do.  Thoughts?

Comment (emilylaguna) [reply WordPress/Classes/System/WordPressAppDelegate.swift]: After a discussion with @frosty about this issue on Slack he came up with a fix that does not involve removing this line.

Comment (emilylaguna) [reply WordPress/Classes/System/WordPressAppDelegate.swift]: After poking around some more I realized that when configuring the appearance for the `LightNavigationController` we weren't setting the `titleTextAttributes` at all, adding that line in fixes this issue.

**Pull request merged**